### DISCUSSION TITLE
Resizable Textarea

### DISCUSSION THREAD

**Discussion by Welfordian** [mantinedev/mantine#4577] [Feature requests]
How come there isn't a prop to make the `Textarea` component resizable?
I see that it adds `resize: none;` to the stylesheet, but what if I don't want that?
I understand I can override the styles, but surely a prop would be better for this? Perhaps `resizable`.

Unless I'm missing something in the docs and this is already possible. Though, I did try searching quite a bit.

Thanks!

**Comment by rtivital** [comment]
What's wrong with overriding styles? It is just one line of code – `<Textarea styles={{ input: { resize: 'both' } }} />`

**Comment by Welfordian** [comment]
That _is_ true, however, native textarea is resizable by default. Why does Mantine `Textarea` default to no resize?

**Reply by Welfordian** [reply]
I also think it's just a little bit confusing. If it's default is not being resizable, I'd expect a prop to enable this, not to have to override styles.
